### DISCUSSION TITLE
Device bringup

### DISCUSSION THREAD

**Discussion by enduring78** [Divested-Mobile/DivestOS-Build#214] [Q&A] [answered]
Hi everyone, where can I suggest/request a device bringup? 

The device I have already has LineageOS builds, but I'd rather have DivestOS installed because of the extra security features.

**Comment by SkewedZeppelin** [comment, marked as answer]
https://github.com/Divested-Mobile/DivestOS-Build/issues/68

**Reply by enduring78** [reply]
Thanks, posted a comment. How long does it usually take to port DivestOS, it it happens at all?
